src/AssemblerHttpServer-Alignments.cpp

```diff
@@ -1182,6 +1183,255 @@ void Assembler::computeAllAlignments(
 }
 
 
+void Assembler::sampleReads(vector<OrientedReadId>& sample, uint64_t n){
+    sample.clear();
+
+    while (sample.size() < n) {
+        // Randomly select a read in the read set
+        uint16_t readId = uint16_t(rand() % reads.size());
```

Comment:
A `uint16_t` can hold a max value of 65536. So this sampling will always choose from the first 65536 reads. Is that intentional? `OrientedReadId` takes a `uint32_t` readId.

Reply:
Hmm definitely a mistake, i thought my compiler was giving me conversion warnings with uint32_t but I must have misread it.